src/shared/components/palette/date/write-date-field.html

```diff
@@ -3,13 +3,14 @@
   <fieldset>
     <legend>
       <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel }}</span>
-      <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-      <span class="error-message" *ngIf="dateControl && dateControl.errors && dateControl.touched">{{dateControl.errors | ccdFirstError}}</span>
     </legend>
-
+    <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+    <span class="error-message" *ngIf="dateControl && dateControl.errors && dateControl.touched">{{dateControl.errors | ccdFirstError:caseField.label}}</span>
```

Comment:
Why the need for the `dateControl` check here? And if this is an issue, shouldn't it also be in the `[isInvalid]` binding, below?

Reply:
here by this dateControl check; system shows the error message on top of the control and with Invalid field we highlight the date individual control in red when there is error that is day, month and year input